api/models/v0.3/modelCard.js

```diff
@@ -39,7 +40,7 @@ const CardValidator = () => {
       card.errors["username"] = ["Username is not available or valid"];
     }
 
-    if (card.email && !/\A[^@]+@[^@]+\z/.test(card.email)) {
+    if (card.email && !/^[^@]+@[^@]+$/.test(card.email)) {
```

Comment:
Why the change?

Reply:
Js regex seems to be slightly different than the ruby version.

Reply:
Yeah, those look like old-time Perl regexes to me.